src/tenant/models.py

```diff
@@ -60,9 +60,9 @@ class Tenant(TenantMixin):
     created_on = models.DateField(auto_now_add=True)
     owner_full_name = models.CharField(
         max_length=255, blank=True, null=True,
-        help_text="The owner of this deck."
+        help_text="DEPRECATED: The owner of this deck."
     )
-    owner_email = models.EmailField(null=True, blank=True)
+    owner_email = models.EmailField(null=True, blank=True, help_text="DEPRECATED: Primary e-mail address.")
```

Comment:
Please use this:
"DEPRECATED: the verified email address of the Deck Owner (set in each deck's Site Config) will be used. This field will be removed in a future update"

Reply:
help_text was update as requested, see https://github.com/bytedeck/bytedeck/pull/1457/commits/29530aa0055fb51100452fc0695cbf6db1fb2b16